webpack.config.prod.js

```diff
@@ -29,6 +29,7 @@ export default {
     loaders: [
       {test: /\.js$/, include: path.join(__dirname, 'src'), loaders: ['babel']},
       {test: /\.(jpe?g|png|gif|svg)$/i, loaders: ['file']},
+      {test: /\.ico$/, loader: 'file-loader?name=[name].[ext]'},
```

Comment:
@jforaker I will suggest that you `{test: /\.ico$/, loader: 'file?name=[name].[ext]'}` instead since the latest webpack complains about adding the loader extension to loaders.